github/ipfs.yml

```
@@ -192,7 +192,6 @@ members:
     - protocollabsit-readonly
     - ratoshniuk
     - raulk
```

I think `reidlw` didn't accept the previous invite.
```suggestion
    - raulk
    - reidlw
```